truss/templates/server/model_wrapper.py

```diff
@@ -347,7 +343,15 @@ def load(self):
                 self._logger.exception("Exception while loading model")
                 self._status = ModelWrapper.Status.FAILED
 
-    def _load_impl(self):
+    async def start_load(self):
+        if self.should_load():
+            asyncio.create_task(self.load())
```

Comment:
What happens if there's an exception in load? Don't we need to await that properly here?

Reply:
if you look in the definition of load, we catch exceptions there. We don't need to await on that here, similar to how we were doing things before, we do this `shutdown_if_load_fails` thing:

https://github.com/basetenlabs/truss/blob/c0f4644778a86088e8be2810e8d145a1c70ba2e7/truss/templates/server/truss_server.py#L266

so if load fails, we mark the status as failed, and then that other task shuts down the server.